include/revng/ADT/GenericGraph.h

```diff
@@ -1527,6 +1527,37 @@ struct GraphTraits<llvm::Inverse<T *>> {
   static NodeRef getEntryNode(llvm::Inverse<NodeRef> N) { return N.Graph; };
 };
 
+// Undirected - This class is used as a marker class to tell the graph iterator
+// to treat the underlying graph as an undirected one
+template<class GraphType>
+struct Undirected {
```

Comment:
Add a `TODO` about moving this to `Undirected.h` and offering a default implementation of `GraphTraits<Undirected<T>>` for any `T` for which you have both `GraphTraits<T>` and `GraphTraits<Inverse<T>>` (what I was suggesting during the rev.ng hour).

Reply:
I added a TODO explaining the situation [here](https://github.com/revng/revng/blob/d25a140596ffda169b22a0334654fc7bb27b028d/include/revng/ADT/GenericGraph.h#L1530). Do you want me to implement this change right away?

Reply:
No, thanks!